portfolio/src/main/webapp/script.js

```diff
@@ -1,7 +1,10 @@
+/** Scrolls to the top of the page when the button is clicked.*/
 function scrollUp() {
   window.scrollTo({top: 0, behavior: 'smooth'});
 }
 
+/**Fetches limit for number of comments shown on page and then adds the
+ * appropriate amount of comments to the UI.*/
```

Comment:
Format this multi-line jsdoc comment like the other one you've done below: with the /** on one line by itself, then each consequent line starting with a single asterisk, and then the final line consisting only of */